### PR TITLE
feature/date-pickers

### PR DESCRIPTION
**Pull request by rajvimalc**
Date Picker Plugin for Jenkins. This Allows user to enter date using a date picker and/or timestamp picker.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
     <groupId>me.leejay.jenkins</groupId>
     <artifactId>date-parameter</artifactId>
-    <version>0.0.5-SNAPSHOT</version>
```
Comment (jeffpearce) [inline pom.xml]: You shouldn't change this as part of a PR - it gets updated as part of the release process

File: src/main/java/me/leejay/jenkins/dateparameter/DateParameterDefinition.java [hunk 2]
```diff
@@ -42,20 +44,23 @@ public String getDescription() {
 
     /**
      * Called from jelly
+     * @return String
```
Comment (jeffpearce) [inline src/main/java/me/leejay/jenkins/dateparameter/DateParameterDefinition.java]: This should describe the actual thing being returned, not the type

File: src/main/java/me/leejay/jenkins/DateParameterValue.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package me.leejay.jenkins.dateparameter;
+package me.leejay.jenkins;
```
Comment (jeffpearce) [inline src/main/java/me/leejay/jenkins/DateParameterValue.java]: If there are any other plugins using anything in here, I think that moving these might break them.

Comment (rajvimalc) [reply src/main/java/me/leejay/jenkins/DateParameterValue.java]: Since version will be upgraded to 0.0.5, should not be a problem.

Comment (rajvimalc) [reply src/main/java/me/leejay/jenkins/DateParameterValue.java]: Okay, reverted to old state.

**Comment by jeffpearce** [comment]
This looks OK to me, but I'm not very familiar with this plugin. @leejaycoke - can you take a look?

**Comment by leejaycoke** [comment]
@rajvimalc Thank you PR.
Unfortunately this PR cannot be merged.
You have written good code. But that's not for the purpose of the plugin.

-This plugin needs to be functionally advanced rather than visual.
-Simple visual features unnecessarily increase users' choices.

**Pull request closed without merging**